### DISCUSSION TITLE
Playbooks, plays and Elastalert

### DISCUSSION THREAD

**Discussion by csanate** [Security-Onion-Solutions/securityonion#5867] [Unsupported Versions] [answered]
I am trying to understand the current correct way to create alerts for specific items in the system. I am on 2.3.80.

I have manually created Elastalert YAML files on my manager in /opt/so/rules/elastalert, included SMTP configuration, etc. This works good but is a bit cumbersome to use. 

As I read thru the docs on Playbooks, I see this can create Elastalert rules based on the play. How would we go about setting up an e-mail alert for this? I don't think Sigma has a provision for this as it's supposed to be portable, but, perhaps I am mistaken?

For example, in my manually created Elastalert YAML file, I can insert the following to be alerted via e-mail when my alert fires... can I do something similar in the Playbooks interface by modifying an existing play?

```
email:
 - redacted@redacted.com
smtp_host: "redacted.com"
smtp_port: 25
smtp_ssl: false
from_addr: "redacted@redacted.com"
smtp_auth_file: "smtp_auth_file.txt"
```

Thanks!

**Comment by defensivedepth** [comment, marked as answer]
Emailed alerts is not supported in Playbook - it will only generate an alert and make it viewable in the SOC Alerts interface.
